tests/nvme_test.py

```diff
@@ -228,6 +231,19 @@ def get_ncap(self):
         print(ncap)
         return int(ncap)
 
+    def get_ocfs(self):
+        """ Wrapper for extracting optional copy formats supported
+            - Args:
+                - None
+            - Returns:
+                - Optional Copy Formats Supported
+        """
+        pattern = re.compile("^ocfs[ ]+: 0x[0-9a-fA-F]+", re.IGNORECASE)
```

Comment:
The one-element character class `[ ]` is kinda strange. Could you also add a `$` at the end to ensure the whole line matches? And I don't think `re.IGNORECASE` should be necessary.

Reply:
```
-        pattern = re.compile("^ocfs[ ]+: 0x[0-9a-fA-F]+", re.IGNORECASE)
+        pattern = re.compile(r'^ocfs\s+: 0x[0-9a-fA-F]+$')
```